pytensor/tensor/nlinalg.py

```diff
@@ -543,13 +546,23 @@ class SVD(Op):
     def __init__(self, full_matrices: bool = True, compute_uv: bool = True):
         self.full_matrices = bool(full_matrices)
         self.compute_uv = bool(compute_uv)
+        if self.compute_uv:
+            if self.full_matrices:
+                self.gufunc_signature = "(m,n)->(m,m),(n),(n,n)"
+            else:
+                self.gufunc_signature = "(m,n)->(m,k),(k),(k,n)"
+        else:
+            self.gufunc_signature = "(m,n)->(n)"
```

Comment:
I think this is the correct signature? IIUC the output is the min of `(m, n)`?
```suggestion
            self.gufunc_signature = "(m,n)->(k)"
```